person_capture/gui_app.py

```
@@ -4912,156 +5291,67 @@ def save_hit(c, idx):
-                                interval=1.5,
+                        try:
+                            cur_w = max(1.0, float(cx2 - cx1))
+                            cur_h = max(1.0, float(cy2 - cy1))
+                            cx1, cy1, cx2, cy2 = self._ratio_crop_containing_box(
+                                protect_box,
+                                ratio_str,
+                                (bx1, by1, bx2, by2),
+                                anchor=((cx1 + cx2) * 0.5, (cy1 + cy2) * 0.5),
+                                min_size_xy=(cur_w, cur_h),
+                            )
+                        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)